app/controllers/lettings_logs_controller.rb

```diff
@@ -111,6 +111,31 @@ def update_logs
     end
   end
 
+  def delete_logs_list
+    log_ids = params.require(:log_ids).map(&:to_i)
+    selected_ids = params[:selected_ids]&.map(&:to_i)
```

Comment:
Are you sure that you need to map these to integers? AR finders are quite content with strings.